### DISCUSSION TITLE
I keep getting Build and deployment error when I'm trying to publish my website

### DISCUSSION THREAD

**Discussion by bicmfcs** [alshedivat/al-folio#1309] [Q&A]
Hi, 

I can locally see the webpage I created using alfolio, and I pushed it to the [repository](https://github.com/bicmfcs/bicmfcs.github.io) but I keep getting [pages build and deployment](https://github.com/bicmfcs/bicmfcs.github.io/actions/runs/4585370329) error. 
and it says :
`github-pages can't satisfy your Gemfile's dependencies.`

Thanks for your help

**Comment by george-gca** [comment]
Did you manage to fix it?

**Reply by augustfi** [reply]
I have the same issue

**Reply by george-gca** [reply]
Have you thoroughly followed the deployment part in [README](https://github.com/alshedivat/al-folio#deployment), specially the `To enable automatic deployment` section?
